package/README.md

```diff
@@ -1,44 +1,81 @@
-# GOV.UK Frontend - All components
+# GOV.UK Frontend
 
-All of the components in GOV.UK Frontend, in a single package.
+GOV.UK Frontend contains the code you need to start building a user interface
+for government platforms and services.
 
-## Guidance
+See live examples of GOV.UK Frontend components, and guidance on when to use
+them in your service, in the [GOV.UK Design
+System](https://govuk-design-system-production.cloudapps.digital/).
 
-Find out when to use the Back link component in your service in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital).
+## Contact the team
 
-## Dependencies
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
+team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
 
-To consume all components you must be running npm version 5 or above.
+## Requirements
 
-## Installation
+To use GOV.UK Frontend with [npm](https://www.npmjs.com/) you must:
 
-    npm install --save govuk-frontend
+Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM. The minimum version of Node required is 8.6.0.
 
-## Requirements
+## Quick start
+
+There are 2 ways to start using GOV.UK Frontend in your app.
+
+Once installed, you will be able to use the code from the examples in the
+[GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/)
+in your service.
+
+### 1. Install with npm (recommended)
+
+We recommend [installing GOV.UK Frontend using node package manager
+(npm)](docs/installation/installing-with-npm.md).
```

Comment:
This link is broken – and when this is published to npm we can't reference another path relatively at all. Suggest making this a link including github.com (like we have for 'More details on importing styles')?

There's a few other links in this file that are also broken – I won't comment on them all.